### PR TITLE
feat(js): LANZ-2248 improved workflow to support breaking change excl…

### PR DESCRIPTION
**Pull request by cangulomascarell**
…amation

### CODE REVIEW

**Comment by cangulomascarell** [comment]
execution example: https://github.com/ohpen/platform-repository-for-testing/actions/runs/2154871430

File: custom-conventional-commits-accepted.json [hunk 1]
```diff
@@ -11,20 +11,8 @@
         "commitType": "feat",
         "releaseType": "minor"
     },
```
Comment (cangulomascarell) [inline custom-conventional-commits-accepted.json:13]: deleted unused commit types, those were for testing purposses

File: generate-version-and-release-notes.js [hunk 2]
```diff
@@ -135,26 +137,44 @@ function getUpdatedVersion(version, changes) {
...
+
+    if (validPrefix) {
+      result.type = breaking ? "major" : commitTypes.find(x => x.commitType == type).releaseType
+      result.content = subject.trim()
+      if (scope)
+        result.content = `${scope.trim()}: ${result.content}`
+    } else {
+      result.type = "none"
+      result.content = line.trim()
+    }
```
Comment (cangulomascarell) [inline generate-version-and-release-notes.js:161]: changelog example:
https://github.com/ohpen/platform-repository-for-testing/blob/main/CHANGELOG.md

image

**Comment by cangulomascarell** [comment]
once this PR is approved: https://github.com/ohpensource/ensure-conventional-commits-gh-action/pull/5

the continuous integration check that is failing will pass

**Review by jcajkovic — APPROVED** [review]

**Review by jstehlik-ohpen — APPROVED** [review]

**Pull request merged**